odxtools/compumethods/createanycompumethod.py

```diff
@@ -52,9 +52,10 @@ def _parse_compu_scale_to_linear_compu_method(
     offset = computation_python_type(next(nums).text)
     factor_el = next(nums, None)
     factor = computation_python_type(factor_el.text if factor_el is not None else "0")
-    if coeffs.find("COMPU-DENOMINATOR/V") is not None:
-        kwargs["denominator"] = float(coeffs.findtext("COMPU-DENOMINATOR/V"))
-        assert kwargs["denominator"] > 0
+    denominator = 1.0
+    if (string := coeffs.findtext("COMPU-DENOMINATOR/V")) is not None:
+        denominator = float(string)
+        assert denominator != 0
```

Comment:
use a warning

Reply:
other than this, LGTM

Reply:
ok fixed: de74a6b3211566d0abc5b40da328ad28aea48c8d. Let's get this merged...